pandas/_typing.py

```diff
@@ -0,0 +1,4 @@
+from pathlib import Path
```

Comment:
can you add a test that asserts this in pandas/tests/types/test_api.py

Reply:
Yea no problem. So the existing test(s) exclude any privately named modules:

https://github.com/pandas-dev/pandas/blob/27927056b368fa8c9501c4ec8729546dbe98068c/pandas/tests/api/test_api.py#L14

Are you asking to revisit that logic or simply add a test with this as an exception to make sure it lives there?

Reply:
oh i c. nvm then. though I think we should actually check and lock down all modules (so add back the private ones). in a new PR / issue though.